website/docs/migrations/11-12.md

```diff
@@ -218,3 +218,40 @@ const projectConfig = {
 	},
 };
 ```
+
+### Update ES Lint config
+
+**Migration time: ~10min.**
+
+Depending on the state the project is in, you might need to allocate some time for fixing errors and warnings after updating the coding standards.
+
+ESLint configuration has been updated in Frontend libs 12 and a new build config was added.
+
+To implement the changes, follow these steps:
+
+1. Delete the `.eslintrc.js`  and `.eslintignore` files from the theme root (if they exist)
+
+2. Add `eslint.config.mjs` file to the theme root with the following content:
+```js
+import config from '@eightshift/frontend-libs/linters/eslint.config.mjs';
+
+export default [{
+    ignores: ["public/**/*.js"],
+}, ...config];
+```
+
+3. Add a `.swcrc` file to theme root with the following content:
+```json
```

Comment:
remove this part and replace it with the link from the f libs
https://github.com/infinum/eightshift-frontend-libs/blob/main/.swcrrc 

we will add the necessary files to the repo as they are missing at this point